.travis.yml

```diff
@@ -84,6 +84,9 @@ notifications:
     on_start:   change
 
 before_install:
+# Download codespell to test PR for spelling mistakes
+- pip install --user --upgrade git+https://github.com/lucasdemarchi/codespell.git
+- source ${TRAVIS_BUILD_DIR}/spellcheck_code.sh
```

Comment:
is line 89 really needed?

Reply:
@sasobadovinac what other way do you want to invoke the script?

Reply:
as I understand you are just running it twice first at line 89 and a second time at line 240